torch/csrc/autograd/variable.h

```diff
@@ -20,6 +20,8 @@
 
 namespace torch { namespace autograd {
 
+using variable_list = std::vector<at::Tensor>;
```

Comment:
In torch/csrc/autograd/cpp_hook.h you replaced `variable_list` by `std::vector<at::Tensor>`.
But here you change it. Is there a reason for these two different behaviors?

Reply:
No. I started off by replacing occurrences of `variable_list`, but then changed my mind and added back the alias (and didn't clean up the old sites). We should probably just eliminate the `variable_list` alias entirely from the codebase, but not this PR please!

Reply:
I think removing `variable_list` is a BC though no? Isn't it used in some of our tutorials?

Reply:
Yeah, I didn't delete it in the end, there should be an alias definition everywhere there used to be.